llvm-node.d.ts

```diff
@@ -199,6 +199,8 @@ declare namespace llvm {
     private constructor();
 
     readonly value: number;
```

Comment:
Should we add a check to get that throws if the value is larger than what JS supports?